tasks/6to5.js

```diff
@@ -11,7 +11,12 @@ module.exports = function (grunt) {
 
 			var res = to5.transformFileSync(el.src[0], options);
 
-			grunt.file.write(el.dest, res.code);
+			var sourceMappingURL = '';
+			if (res.map) {
+				sourceMappingURL = grunt.util.linefeed + '//# sourceMappingURL=' + el.dest.split('/').pop() + '.map';
```

Comment:
`grunt.util.linefeed == \n`, no?

The `el.dest` could be a path, but you only want the filename part for the sourceMappingURL directive. Seems uglify is doing it that way, anyway?

So:
```javascript
module.exports = {
    options: {
        sourceMap: true
    },
    dist: {
        files: {
            'some/path/6to5/dest.js': 'some/other/path/source.js'
        }
    }
};
```
ouputs `some/path/6to5/dest.map.js` and adds `//# sourceMappingURL=dest.js.map

Is this the wrong way to do it? Should it output the whole path?